config_test.go

```diff
@@ -128,20 +129,43 @@ func x509Cert(c, k []byte) tls.Certificate {
 	return cert
 }
 
+func createFile(namePattern string, content []byte, mode os.FileMode) (filename string, err error) {
```

Comment:
I recommend removing `mode` per other comment

```suggestion
func createFile(namePattern string, content []byte) (filename string, err error) {
```